lua/example_output_script.lua

```diff
@@ -0,0 +1,46 @@
+-- This is a simple example script to show what you can do with lua output scripts.
```

Comment:
the name of this script was 'fast.lua' because it implements an output like our built-in 'fast.log', so I think the name should continue to reflect this.

Reply:
okay, makes sense